README.rst

```diff
@@ -261,10 +261,45 @@ To save memory, both the ``Ok`` and ``Err`` classes are ‘slotted’,
 i.e. they define ``__slots__``. This means assigning arbitrary
 attributes to instances will raise ``AttributeError``.
 
+The ``as_result()`` decorator can be used to quickly turn ‘normal’
+functions into ``Result`` returning ones by specifying one or more
+exception types:
 
-FAQ
--------
+.. sourcecode:: python
+
+    @as_result(ValueError, IndexError)
```

Comment:
Where's the IndexError in this example?

Reply:
I think we can either update the example to showcase multiple exceptions or just remove it?

Reply:
i included multiple exceptions to showcase that more than one can be passed.

i've updated the example to showcase the happy path, and also 3 different exceptions, 2 of which are caught, and 1 of which is not.

Reply:
👍